src/azure-cli/azure/cli/command_modules/keyvault/_params.py

```diff
@@ -389,7 +389,6 @@ class CLIJsonWebKeyCurveName(str, Enum):
     with self.argument_context('keyvault key restore', arg_group='Restoring keys from storage account') as c:
         c.argument('token', options_list=['--storage-container-SAS-token', '-t'],
                    help='The SAS token pointing to an Azure Blob storage container')
-        c.argument('backup_folder', help='Name of the blob container which contains the backup')
```

Comment:
we need to sync with SDK team to understand why the breaking change. this IS a breaking change to CLI users. thanks for the email.

Reply:
how about we keep this param in CLI for a while with deprecation message, explain it will be auto retrieved from storage-resource-uri in future, the param itself just do nothing.

Reply:
I sent back an email a little while ago, but to make sure I state it here: there don't need to be any breaking changes in the CLI. The SDK now accepts a `backup_storage_url` that's a concatenation of what used to be provided as `backup_storage_uri` and `folder_name`, but this full URL can be generated by the CLI. [This](https://github.com/Azure/azure-sdk-for-python/commit/1d177a51b165618e92f0657dc2ef2d73434398fb#diff-2d6f74576bcbe510221dbdbc54c0e797d3baf5541dde99e5d356595b4ad82eb4L49) is the PR where the change was made on the SDK's side, for reference.

Reply:
Thanks @yungezz @mccoyp, we don't need command interface breaking change now.